### PR TITLE
Feature/add support for new venue info endpoints

### PR DESCRIPTION
**Pull request by alexjohnson505**

### CODE REVIEW

File: src/venue/resource.js [hunk 1]
```diff
@@ -127,12 +127,20 @@ angular
...
       },
-      
       updateInfo: {
         method: 'PUT',
-        url: endpoint + '/info'
+        url: endpoint + '/info/:key'
+      },
+      createInfo: {
+        method: 'POST',
+        url: endpoint + '/info/'
```
Comment (mattetre) [inline src/venue/resource.js]: should drop trailing `/`

File: src/venue/service.js [hunk 2]
```diff
@@ -155,17 +155,42 @@ angular
...
-        return Venue.listInfo(options).$promise;
+      VenueService.prototype.readInfo = function readInfo(venueId, key, options) {
+        if (!venueId) throw new Error('venueId is required');
+        if (!key) throw new Error('key is required');
+
+        options = options || {};
+
+        return Venue.readInfo({
+          id : venueId,
+          key : key,
```
Comment (mattetre) [inline src/venue/service.js]: drop trailing `,`

File: src/venue/service.js [hunk 3]
```diff
@@ -155,17 +155,42 @@ angular
...
+        if (!options) throw new Error('options is required');
+        if (!options.key) throw new Error('options.key is required');
+
+        return Venue.createInfo({
+          id : venueId,
+        }, options).$promise;
+      };
 
-        return Venue.listInfo(options).$promise;
+      VenueService.prototype.readInfo = function readInfo(venueId, key, options) {
```
Comment (mattetre) [inline src/venue/service.js]: what would go in the `options`?

File: src/venue/service.js [hunk 4]
```diff
@@ -155,17 +155,42 @@ angular
...
         if (!venueId) throw new Error('venueId is required');
-        if (!options) throw new Error('options is required');        
+        if (!key) throw new Error('key is required');
+
+        options = options || {};
 
         return Venue.updateInfo({
-          id: venueId,
+          id : venueId,
+          key : key,
```
Comment (mattetre) [inline src/venue/service.js]: trailing `,`

**Comment by mattetre** [comment]
Looks good. Would be awesome to add jsdocs to the new functions so we can generate documentation and so people know what data they can pass into the options parameters

**Comment by alexjohnson505** [comment]
Added JSDocs to the new functions. Addressed comments. Added /release folder.

**Pull request merged**